roles/pulp_content_guard_rbac/tasks/main.yml

```diff
@@ -0,0 +1,68 @@
+---
+
+- name: Get RBAC content guard list
+  uri:
+    url: "{{ pulp_rbac_cg_url }}"
+    user: "{{ pulp_admin_username }}"
+    password: "{{ pulp_admin_password }}"
+    method: GET
+    status_code: 200
+    force_basic_auth: true
+  no_log: true
+  register: rbac_cg_list_result
+
+- name: Set fact cgnames
+  set_fact:
+    cgnames: "{{ (cgnames | default([])) + [item.name] }}"
+  with_items: "{{ rbac_cg_list_result.json.results }}"
+
+- name: Create RBAC content guards
+  uri:
+    url: "{{ pulp_rbac_cg_url }}"
+    user: "{{ pulp_admin_username }}"
+    password: "{{ pulp_admin_password }}"
+    force_basic_auth: true
+    method: POST
+    status_code: 201
+    body:
+      name: "{{ item.name }}"
+    body_format: form-urlencoded
+  loop: "{{ pulp_content_guards_rbac_present | default([], true) }}"
```

Comment:
No need for default filter when there is a role default.

Reply:
I'll test it again when I get my pulp VM back, but this was to fix an error when pulp_content_guards_rbac_present was defined but not empty

Reply:
As in just state in the documentation that it needs to be a list, or go from `default([], true)` to `default([])`?

Reply:
Yes - in the docs. We already have a role default, so no need for a default filter.